server/openejb-cxf-rs/src/main/java/org/apache/openejb/server/cxf/rs/CxfRSService.java

```diff
@@ -224,6 +224,7 @@ private void initCxfProviders(final Bus bus) {
 
             final Collection<Object> defaults = new ArrayList<>();
             for (final String provider : asList(
+                    "org.apache.johnzon.jaxrs.jsonb.jaxrs.JsonbJaxrsProvider",
```

Comment:
can't be added with TomEEJohnzonProvider since one of both will never be used, maybe implement a custom AccessMode merging johnzon and jsonb ones and set the resulting provider from it